tests/envs/test_make.py

```diff
@@ -228,13 +230,16 @@ def test_make_render_mode():
         )
 
     # Make sure that an additional error is thrown a user tries to use the wrapper on an environment with old API
-    with pytest.raises(
-        gym.error.Error,
-        match=re.escape(
-            "You passed render_mode='human' although test/NoHumanOldAPI-v0 doesn't implement human-rendering natively."
-        ),
-    ):
-        gym.make("test/NoHumanOldAPI-v0", render_mode="human", disable_env_checker=True)
+    with warnings.catch_warnings(record=True):
```

Comment:
Again, do we want to check the warnings?

Reply:
repeat of the same general issue